random.c

```diff
@@ -1683,6 +1683,11 @@ InitVM_Random(void)
 	rb_define_method(m, "random_number", rand_random_number, -1);
 	rb_define_method(m, "rand", rand_random_number, -1);
     }
+
+#ifdef _FORTIFY_SOURCE
```

Comment:
@ko1 changing this to `#if VM_CHECK_MODE` requires `#include "vm_core.h"` from random.c. Is it acceptable?

Reply:
nevermind, rerouted this issue on https://github.com/ruby/ruby/pull/1947/commits/866f046eccb202276c8e0367454f5f9e0d185594

Reply:
👍